packages/sdk/src/services/index.ts

```diff
@@ -0,0 +1 @@
+export * from './send';
```

Comment:
These index files are not that useful internally. I'd suggest to keep only the src/index.ts for now.

Reply:
removed [c36bf33](https://github.com/Voltz-Protocol/v2-off-chain-monorepo/pull/10/commits/c36bf330f7daf2dab92b10c2d0c1ef4b5e4e1cc0)